conda_package/mpas_tools/scrip/from_mpas.py

```diff
@@ -47,10 +44,13 @@ def scrip_from_mpas(mpasFile, scripFile, useLandIceMask=False):
     sphereRadius = float(fin.sphere_radius)
     on_a_sphere = str(fin.on_a_sphere)
 
+    # change the longitude convention to use positive values [0 2pi]
+    lonCell[lonCell<0.0] += 2.0*np.pi
+    lonVertex[lonVertex<0.0] += 2.0*np.pi
+
```

Comment:
Here and in the corresponding edit below, should there be a check to see if `lonCell` and `lonVertex` are `> 2.0*np.pi`?  I believe the more compact way to ensure that these are really in the intended range would be:
```suggestion
    # change the longitude convention to use positive values [0 2pi]
    lonCell = numpy.mod(lonCell, 2.0*np.pi)
    lonVertex = numpy.mod(lonVertex, 2.0*np.pi)
```
I verified that this works for negative longitudes:
```
$ python
>>> import numpy
>>> numpy.mod(-numpy.pi, 2*numpy.pi)
3.141592653589793
```